patch-kernel/patch_kernel.sh

```diff
@@ -6,7 +6,7 @@ DIFF_DIR=$1
 for ext in diff patch; do
   if ls ${DIFF_DIR}/*.${ext} 1>/dev/null 2>&1; then
     for file in ${DIFF_DIR}/*.${ext}; do
-      if patch --dry-run -p1 -s < "${file}" 2>/dev/null; then
+      if patch --dry-run -N --silent -p1 -s < "${file}" 2>/dev/null; then
```

Comment:
any reason for the -N here?

Reply:
From man:

> -N  or  --forward
>           When a patch does not apply, patch usually checks if the patch
>           looks like it has been applied already by trying to reverse-
>           apply the first hunk.  The --forward option prevents that.
>           See also -R.

This is to avoid prints like this in the job logs:

> 1 out of 1 hunk ignored
> Reversed (or previously applied) patch detected!  Assume -R? [n] 
> Apply anyway? [n] 

IIRC `--silent` does not prevent this prompt.